MultiServer.py

```diff
@@ -1837,6 +1837,9 @@ def update_client_status(ctx: Context, client: Client, new_status: ClientStatus)
     if current != ClientStatus.CLIENT_GOAL:  # can't undo goal completion
         if new_status == ClientStatus.CLIENT_GOAL:
             ctx.on_goal_achieved(client)
+            if all(player in ctx.client_game_state and ctx.client_game_state[player] == ClientStatus.CLIENT_GOAL
+                   for player in ctx.player_names if player[0] == client.team and player[1] != client.slot):
```

Comment:
```suggestion
                   for player in ctx.player_names
                   if player[0] == client.team and player[1] != client.slot):
```
I think it's slightly easier to read and understand with a line break between the iterable and the filter.